vertical-pod-autoscaler/pkg/recommender/model/types.go

```diff
@@ -141,6 +149,29 @@ func resourceAmountFromFloat(amount float64) ResourceAmount {
 	}
 }
 
+// humanizeMemoryQuantity converts raw bytes to human-readable string using binary units (KiB, MiB, GiB, TiB) with no decimal places.
+func humanizeMemoryQuantity(bytes int64) string {
```

Comment:
Does it make sense to write some tests only for this function?

Reply:
Adding tests is always beneficial in my opinion. We can add more tests for the entire `types.go`, but I believe that goes beyond the scope of this PR

Reply:
Agreed. May be add tests for `humanizeMemoryQuantity()` now, and in a later PR add tests for `types.go` ?

Reply:
No problem, I’ll add more tests specifically for that function and open an issue for the full unit test coverage.